clang/test/dpct/warp_cuda11_after.cu

```diff
@@ -0,0 +1,95 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
+// RUN: dpct -out-root %T/warp_cuda11_after %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
+// RUN: FileCheck --input-file %T/warp_cuda11_after/warp_cuda11_after.dp.cpp --match-full-lines %s
+
+#include "cuda.h"
+#include "cuda_runtime.h"
+
+__global__ void reduce_add_sync() {
+  unsigned mask;
+  int val;
+  // CHECK: sycl::reduce_over_group(item_ct1.get_sub_group(), val, sycl::plus<>());
```

Comment:
Should emit warning message to tell user mask not migrated.

Reply:
fixed